src/main/resources/static/style.css

```diff
@@ -2,7 +2,7 @@
 
 
 table {
-	width: 1400px;
+	width: 100%;
```

Comment:
Good idea... a pixel value, like 1400px will not look good on most devices.  Percentage is much more flexible.